clang/lib/CIR/Dialect/Transforms/TargetLowering/RecordLayoutBuilder.cpp

```diff
@@ -478,6 +480,31 @@ void ItaniumRecordLayoutBuilder::layoutFields(const StructType D) {
   }
 }
 
+void ItaniumRecordLayoutBuilder::finishLayout(const StructType D) {
+  // If we have any remaining field tail padding, include that in the overall
+  // size.
+  setSize(std::max(getSizeInBits(), (uint64_t)Context.toBits(PaddedFieldSize)));
+
+  // Finally, round the size of the record up to the alignment of the
+  // record itself.
+  uint64_t UnpaddedSize = getSizeInBits() - UnfilledBitsInLastUnit;
```

Comment:
still missing a bunch of camelback here, if there are more elsewhere plz fix them too

Reply:
We have clang-tidy rules set up to try to catch casing violations. If you have any way of integrating clang-tidy into your editor (it usually comes along if you set up a clangd LSP, for example), it'll flag all the casing errors and can even auto-fix them, which is pretty convenient.

Reply:
These are from the OG codegen and I wanted to use them as they were there, but I'll update.

Reply:
updated